### PR TITLE
Added Views associated with Models and Controllers I've already created

### PR DESCRIPTION
**Pull request by jkanyi-web**
- Implemented the design from the sneak peek wireframe
- Used methods that I had created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- Added some data to my database, in order to see some posts or comments,

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+  <p>Bio</p>
+  <p><%= @user.bio %></p>
+</div>
+
+<% @user.posts.each_with_index do |post, index| %>
+  <div class="jumbotron p-3 shadow">
+    <p>Post#<%= index + 1 %></p>
+    <p><%= post.title %></p>
+  </div>
+<% end %>
```
Comment (KanzaTahreem) [inline app/views/users/show.html.erb]: - [x] Good job so far 👏 However, kindly make sure to use the `#recent_posts` method to render the user's 3 most recent post mentioned in the [project requirements] and according to the provided wireframe as well 🤓⛏
- [x] Also, please make sure to display the post text as we have post title. Currently, only title is visible, let's also add the text ✨💪 something like below 👇


```ruby
<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
```

- [x] Also, Kindle make sure to display the comments counter and likes counter associated with each post as displayed on the wireframe ✨ you can use something like below 👇

```ruby
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>
```

Current UI | Wireframe
---| ---|
image | image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+    <p class="right">Comments: <%= post.comments.count %> Likes: <%= post.likes.count %></p>
+    <% if post.comments.any? %>
+      <div class="jumbotron shadow p-5">
+        <% post.comments.each do |comment| %>
+          <p><%= comment.user.name %>: <%= comment.text %></p>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+<% end %>
```
Comment (KanzaTahreem) [inline app/views/posts/index.html.erb]: - [x] Great effort so far 👏✨ However, kindly make sure to use the `recent_comments` method present in `post.rb` to render 5 most recent comments on post index page as [displayed](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAAB5V373UCMRN55B3KJOJG2WAZLQVKZQ)

Current UI | Wireframe
---| ---|
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @jkanyi-web 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @jkanyi-web,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**